### ISSUE TITLE
New-HPOVLogicalEnclosure returns string format error

### ISSUE THREAD

**Issue by daveolker** [HewlettPackard/POSH-HPOneView#287]
Please fill in as much information as possible to help resolve your issue.

### Expected Behavior

Expect New-HPOVLogicalEnclosure to complete without error.

### Actual Behavior

Call to New-HPOVLogicalEnclosure returns the following error:

```PowerShell
Error formatting a string: Input string was not in a correct format..
At C:\Program Files\WindowsPowerShell\Modules\HPOneView.310\HPOneView.310.psm1:31764 char:5
+ ...             "**[{0]}]** Synergy Frame object: {1}" -f $MyInvocation.Invoc ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: ([{0]}] Synergy Frame object: {1}:String) [], RuntimeException
    + FullyQualifiedErrorId : FormatError
```
Looks like there might be an extra ']' character at line 31764.

### Steps to reproduce
```PowerShell
$EG = Get-HPOVEnclosureGroup -Name EG-Synergy-Local
$Encl = Get-HPOVEnclosure -Name Synergy-Encl-1
New-HPOVLogicalEnclosure -EnclosureGroup $EG -Name LE-Synergy-Local -Enclosure $Encl | Wait-HPOVTaskComplete
```
### Version Information
HPE OneView PowerShell Library Version (`Get-HPOVVersion`):
```PowerShell
LibraryVersion Path                                                    
-------------- ----                                                    
3.10.1416.3042 C:\Program Files\WindowsPowerShell\Modules\HPOneView.310
```
HPE OneView Appliance Version:  3.10.05-0302465

Output from `$PSVersionTable` on your Windows Host:
```PowerShell
Name                           Value                                                                                                                    
----                           -----                                                                                                                    
PSVersion                      5.1.14393.1532                                                                                                           
PSEdition                      Desktop                                                                                                                  
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                                                                                                  
BuildVersion                   10.0.14393.1532                                                                                                          
CLRVersion                     4.0.30319.42000                                                                                                          
WSManStackVersion              3.0                                                                                                                      
PSRemotingProtocolVersion      2.3                                                                                                                      
SerializationVersion           1.1.0.1                         
```

**Comment by ChrisLynchHPE** [comment]
Thanks for reporting this.  I will have it fixed in the next update.

**Comment by ChrisLynchHPE** [comment]
This is now fixed in [Release 3.10.1471.1581](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v3.10.1471.1581), which has also been published to the [PowerShellGallery](https://www.powershellgallery.com/packages/HPOneView.310).

**Comment by ChrisLynchHPE** [comment]
Closing due to no further activity.
